lib/dokku_cli/ps.rb

```diff
@@ -10,6 +10,30 @@ def ps
       run_command "ps #{app_name}"
     end
 
+    desc "ps:scale PROC1=SCALE1 [PROC2=SCALE2 ...]", "Scale one or more Procfile processes"
+    def ps_scale(*args)
+
+      args = args.map{|arg|
```

Comment:
Could you make this a block and use [`map!`](http://ruby-doc.org/core-2.4.0/Array.html#method-i-map-21) (Which also gets rid of the assignment) .
```ruby
args.map! do |arg|
  ...
end
```